client/src/components/common/boost/BoostResults.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { BoostResultT } from 'types/boost';
+import { Card, Accordion, Table } from 'react-bootstrap';
+
+export interface BoostResultProps {
+  results: BoostResultT | null;
+}
+
+/**
+ * Displays the Maximum achievable speed and the power plan (the distance and the associated recommended power)
+ *
+ * @param props BoostResultProps
+ * @returns JSX Element
+ */
+export default function BoostResult(props: BoostResultProps) {
```

Comment:
Is this component called `BoostResult` or `BoostResults`? There's a discrepancy between the definition here, the filenames, and the names it is imported as.